src/sidebar.ts

```diff
@@ -683,14 +684,38 @@ export class PanelWebview implements vscode.WebviewViewProvider {
         global.is_chat_streaming = state;
     }
 
+    removeQuestionMarks(filePath: string): string {
+        let cleanedPath = filePath.replace(/\\\?\\/g, '');
+        cleanedPath = cleanedPath.replace(/^\\+/, '');
+        return cleanedPath;
+    }
+
     async startFileAnimation(fileName: string) {
 
         const openFiles = this.getOpenFiles();
+        console.log(`[DEBUG]: opened files: `, openFiles);
         const editor = vscode.window.activeTextEditor;
-        if(!openFiles.includes(fileName)|| !editor) {return;}
-        const document = await vscode.workspace.openTextDocument(fileName);
 
-        const state = estate.state_of_editor(editor, "start_animate for file: " + fileName);
+        const workspaceFolders = vscode.workspace.workspaceFolders;
+        if (!workspaceFolders) {
+            return;
+        }
+        
+        const workspace_path = workspaceFolders[0].uri.fsPath;
+        console.log(`[DEBUG]: workspacePath: `, workspace_path);
+        
+        const relative_path = path.relative(workspace_path, fileName.replace(/\?/g, ''));
+
+        const normalized_path = this.removeQuestionMarks(fileName);
+        const uri = vscode.Uri.file(normalized_path);
```

Comment:
I added a method in this branch, can you check if it works? 
https://github.com/smallcloudai/refact-vscode/pull/154/files#diff-38af48824ce0f12123f276c627dfa072e9caa5606fad3688a8c97bdb356cda7dR687-R692

Reply:
It makes the path being generally valid, but vscode cannot open it anyway
![image](https://github.com/user-attachments/assets/774580e6-a556-433c-8ca2-3ff4edbb73b7)

Reply:
is it because of the `/?/` ?

Reply:
also is the drive part of the path case sensitive?

Reply:
It should not be case sensitive, drive name can be rather `C:/` or `c:/`

Reply:
> is it because of the `/?/` ?

It's because it has not enough backslashes at the beginning, so this path is invalid, but we shouldn't rely on this extended path, since vscode treats them as different files and opens new tab constantly